### PR TITLE
Feature/validation: changes in code that turn the button on and off

### PR DESCRIPTION
**Pull request by stepan-sych**

### CODE REVIEW

File: scripts/scriptCards.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+let elementsCards = [{
+    srcImage: "/image/cascade-7152189_960_720.jpg",
```
Comment (antonsych3) [inline scripts/scriptCards.js:2]: Here are 6 images in your code but in fact you have 7 images. For what?

File: scripts/scriptCards.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        "                <p>" + element.paragraf + "</p>" +
+        "            </div>" +
+        "            <div class=\"category\">" +
+        "                <a href=\"#\"><span>Опис природи</span></a>" +
+        "                <span><i class=\"fa-regular fa-eye\"></i></span>" +
+        "            </div>";
+}
+
+for (let element of elementsCards) {
+    let htmlCode = appendToHtml(element);
```
Comment (antonsych3) [inline scripts/scriptCards.js]: I don't think that we need this variable.

File: scripts/scriptValidation.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    console.log("input");
+    console.log(input.value);
+    input.parentElement.classList.add("error");
+    input.classList.add("error");
+}
+
+const formRemoveError = (input) => {
+    input.parentElement.classList.remove("error");
+    input.classList.remove("error");
+}
```
Comment (antonsych3) [inline scripts/scriptValidation.js]: First of all what does it mean - `form remove error`? Please use correct English expressions. Second - this method just removes a new class. What about `removeErrorClass`?
image

File: scripts/scriptValidation.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+    return /^[0-9,!@#$%^&*)(_/?\-=.`'"|~\]\[₴№;:}{\\><]/.test(value);
+}
+
+const isTelNumValidation = (value) => {
+    return !/^[0-9]*$/.test(value);
+}
+
+const formAddError = (input) => {
+    console.log("input");
+    console.log(input.value);
```
Comment (antonsych3) [inline scripts/scriptValidation.js]: Please remove logs. Use it only for debug purpose.

File: scripts/scriptValidation.js [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+const isNameValidation = (value) => {
+    return /^[0-9,!@#$%^&*)(_/?\-=.`'"|~\]\[₴№;:}{\\><]/.test(value);
+}
+
+const isTelNumValidation = (value) => {
+    return !/^[0-9]*$/.test(value);
+}
+
+const formAddError = (input) => {
```
Comment (antonsych3) [inline scripts/scriptValidation.js]: This method just adds a new class. Use according name `addErrorClass` for the method.
Please check translate for `form add error`

File: scripts/scriptValidation.js [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+        } else if (inputTag.id === "name") {
+            validateField(inputTag, isNameValidation);
+        } else if (inputTag.id === "phone") {
+            validateField(inputTag, isTelNumValidation);
+        }
+    }
+    return isValid;
+}
+
+const isEmailValidation = (value) => {
```
Comment (antonsych3) [inline scripts/scriptValidation.js]: Rename validation methods to something like `isEmailValid`.

File: scripts/scriptValidation.js [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+            isValid = false;
+        } else {
+            formRemoveError(field)
+        }
+    }
+    for (const inputTag of formInputs) {
+        console.log("inputTeg");
+        console.log(inputTag);
+        console.log("formInputs");
+        console.log(formInputs);
```
Comment (antonsych3) [inline scripts/scriptValidation.js]: Remove logs please

File: scripts/scriptCards.js (hunk 2 above)
Comment (stepan-sych) [reply scripts/scriptCards.js]: Deleted variable 'htmlCode'.

File: scripts/scriptCards.js (hunk 1 above)
Comment (stepan-sych) [reply scripts/scriptCards.js:2]: Deleted one image.

File: scripts/scriptValidation.js (hunk 4 above)
Comment (stepan-sych) [reply scripts/scriptValidation.js]: Removed logs.

File: scripts/scriptValidation.js (hunk 6 above)
Comment (stepan-sych) [reply scripts/scriptValidation.js]: Renamed.

File: scripts/scriptValidation.js (hunk 7 above)
Comment (stepan-sych) [reply scripts/scriptValidation.js]: Removed logs.

File: scripts/scriptValidation.js (hunk 5 above)
Comment (stepan-sych) [reply scripts/scriptValidation.js]: Changed

File: scripts/scriptValidation.js (hunk 3 above)
Comment (stepan-sych) [reply scripts/scriptValidation.js]: Changed

**Pull request merged**